src/Analyser/ArgumentsNormalizer.php

```diff
@@ -76,6 +76,64 @@ public static function reorderCallUserFuncArguments(
 		)];
 	}
 
+	/**
+	 * @return array{ParametersAcceptor, FuncCall}|null
+	 */
+	public static function reorderCallUserFuncArrayArguments(
+		FuncCall $callUserFuncCall,
+		Scope $scope,
+	): ?array
+	{
+		$args = $callUserFuncCall->getArgs();
+		if (count($args) < 1 || count($args) > 2) {
```

Comment:
Is call with three arguments really invalid? I'm not sure about it.

Reply:
at least its a fatal error https://3v4l.org/RBJfK